<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorsConfigWidget</class>
 <widget class="QWidget" name="ErrorsConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>563</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_grpGeneral">
     <property name="title">
      <string>Texts</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
      <item row="1" column="1">
       <widget class="QSpinBox" name="kcfg_MaxLines">
        <property name="suffix">
         <string> lines</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="kcfg_MaxCharacters">
        <property name="suffix">
         <string> characters</string>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Maximum number of characters:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Maximum number of lines:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_grpQuickActions">
     <property name="title">
      <string>Absolute Durations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
      <item row="0" column="1">
       <widget class="QSpinBox" name="kcfg_MinDuration">
        <property name="suffix">
         <string> msec</string>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="kcfg_MaxDuration">
        <property name="suffix">
         <string> msec</string>
        </property>
        <property name="minimum">
         <number>2000</number>
        </property>
        <property name="maximum">
         <number>8000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Maximum duration:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Minimum duration:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Relative Durations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="2,1,1">
      <item row="2" column="1">
       <widget class="QSpinBox" name="kcfg_MaxDurationPerCharacter">
        <property name="suffix">
         <string> ms/char</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="alt_MinDurationPerCharacter">
        <property name="suffix">
         <string> ch/sec</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="kcfg_MinDurationPerCharacter">
        <property name="suffix">
         <string> ms/char</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="lab_MinDurationPerCharacter">
        <property name="text">
         <string>Minimum duration:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="alt_MaxDurationPerCharacter">
        <property name="suffix">
         <string> ch/sec</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="lab_MaxDurationPerCharacter">
        <property name="text">
         <string>Maximum duration:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="lab_IdealDurationPerCharacter">
        <property name="text">
         <string>Ideal duration:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="kcfg_IdealDurationPerCharacter">
        <property name="suffix">
         <string> ms/char</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="alt_IdealDurationPerCharacter">
        <property name="suffix">
         <string> ch/sec</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="kcfg_AutoClearFixed">
        <property name="text">
         <string>Automatically clear fixed errors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>kcfg_MaxCharacters</tabstop>
  <tabstop>kcfg_MaxLines</tabstop>
  <tabstop>kcfg_MinDuration</tabstop>
  <tabstop>kcfg_MaxDuration</tabstop>
  <tabstop>kcfg_MinDurationPerCharacter</tabstop>
  <tabstop>kcfg_MaxDurationPerCharacter</tabstop>
  <tabstop>kcfg_AutoClearFixed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
